.tool-versions

```diff
@@ -1,2 +1,3 @@
 elixir 1.9.2
 erlang 22.1.4
+nodejs 10.15.3
```

Comment:
Hm, that's a bit old 👵  A vaguely remember the LTS is `12.something` 😉 

I'll quickly add a more recent version.

Reply:
Oh, sorry :grimacing: . `asdf` suggested me that version and I didn't even check.